QuestionTwo.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Question Two</title>
+        <script>
+        document.addEventListener("DOMContentLoaded", () => {
+        let header = document.getElementById("top");
+        let userInput = window.prompt("Enter your number: ");
```

Comment:
Your number would be a string, it would need to be turned to a number first before comparing if it is even or not